Bonsai.Core.Tests/Workflow.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using Bonsai.Dag;
+using Bonsai.Expressions;
+
+namespace Bonsai.Core.Tests
+{
+    static class Workflow
+    {
+        public static WorkflowCursor New()
+        {
+            var workflow = new ExpressionBuilderGraph();
+            return FromGraph(workflow);
+        }
+
+        public static WorkflowCursor FromGraph(ExpressionBuilderGraph workflow)
+        {
+            return new WorkflowCursor(workflow, null);
+        }
+
+        public static WorkflowCursor FromInspectableGraph(ExpressionBuilderGraph workflow)
```

Comment:
I've removed all unused methods for now. They came up as I was conceiving of the testing infrastructure, as an exercise on whether an API like this might work as a general interface to build workflows in code (which is currently very cumbersome and laborious), but they are currently out-of-scope for testing.